src/Integration.UnitTests/Integration.UnitTests.ruleset

```diff
@@ -1,6 +1,9 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <RuleSet Name="SonarQube" ToolsVersion="14.0">
   <Include Path="..\sonarqube\sonarlint-visualstudiocsharp.ruleset" Action="Default" />
+  <Rules AnalyzerId="SonarAnalyzer.CSharp" RuleNamespace="SonarAnalyzer.CSharp">
```

Comment:
We now have SonarLint and SonarAnalyzer namespaces in our ruleset files. We should re-bind the solution.